### PR TITLE
add e2e performance test

### PR DESCRIPTION
**Pull request by noO0oOo0ob**

### CODE REVIEW

File: e2e_tests/assets/serve.py [hunk 1]
```diff
@@ -20,6 +22,12 @@ def status():
     return "OK"
 
 
+@app.route("/performance", methods=["GET"])
```
Comment (yumiguan) [inline e2e_tests/assets/serve.py:25]: performance -> long_time_service、real_service

File: e2e_tests/test_performance.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            try:
+                res = future.result()
+                print(res)
+            except Exception as e:
+                print(f"Request Error:{e}")
+
+
+def test_performance(lyrebird, mock_server):
+    start_time = time.time()
+    send_request(lyrebird.uri_mock + mock_server.api_performance)
```
Comment (yumiguan) [inline e2e_tests/test_performance.py:39]: lyrebird.uri_extra_mock

File: e2e_tests/conftest.py [hunk 3]
```diff
@@ -78,6 +79,8 @@ def _init_port(self):
         self.api_status = f'http://127.0.0.1:{self.port}/api/status'
         self.uri_mock = f'http://127.0.0.1:{self.port}/mock/'
         self.uri_extra_mock = f'http://127.0.0.1:{self.extra_mock_port}/'
+        self.api_flows = f'http://127.0.0.1:{self.port}/api/flow'
+        self.patch_config_url = f'http://127.0.0.1:{self.port}/api/conf'
```
Comment (yumiguan) [inline e2e_tests/conftest.py:83]: remove

File: e2e_tests/test_performance.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
+import pytest
```
Comment (yumiguan) [inline e2e_tests/test_performance.py:1]: remove unused import

File: e2e_tests/assets/checker.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+from lyrebird import event
+import time
+
+TITLE = 'test_encoder_decoder'
+CHECKER_TIME = 0.5
+
+@event('test')
```
Comment (yumiguan) [inline e2e_tests/assets/checker.py:7]: @event('flow')
remove e2e_tests/assets/performance_event_trigger.py

**Review by yumiguan — APPROVED** [review]

File: e2e_tests/test_performance.py (hunk 2 above)
Comment (noO0oOo0ob) [reply e2e_tests/test_performance.py]: uri_extra_mock have white list, test requests will be filtered out, so keep use uri_mock here.

**Pull request merged**